pgtoolkit/log/parser.py

```diff
@@ -53,7 +73,9 @@ def parse(self, fo):
                 yield record
 
 
-def parse(fo, prefix_fmt, filters=None):
+def parse(
+    fo: Iterable[str], prefix_fmt: str, filters: Optional["NoopFilters"] = None
```

Comment:
Could you please add a linebreak after each parameter?

Reply:
Fixed. Thanks for the suggestion and the review overall.